bindata/network/ovn-kubernetes/self-hosted/ovnkube-master.yaml

```diff
@@ -205,6 +205,17 @@ spec:
           }
           # end of cluster_exists()
 
+          # wait forever doing nothing if this DB is not supposed to be part of the cluster
```

Comment:
Added a note about this. But I think we *do* want to run it in production, not just testing, because who knows when somebody might add a new unexpected master. Better to be resilient I think.